resources/views/components/input/file.blade.php

```diff
@@ -0,0 +1,63 @@
+@props([
+    'name',
+    'label',
+    'value' => null,
+    'error',
+    'filePickedLabel' => 'File Picked:',
+])
+
+<div
+    {{ $attributes->class('flex flex-col gap-1') }}
+    x-data="{ fileName: '', fileSelected: false }"
+>
+    <x-input.label for="{{ $name }}">
+        {{ $label }}
+    </x-input.label>
+
+    <x-button
+        x-show="!fileSelected"
+        class="w-32"
+        type="button"
+        id="file-upload-button"
+        size="sm"
+        @click="$refs.fileInput.click()"
+    >
+        Choose File
+    </x-button>
+
+    <div class="flex flex-col items-start" x-show="fileSelected">
+        <label for="file-upload" class="ml-2 text-xs text-slate-400">
+            {{ $filePickedLabel }}
+        </label>
+        <div class="flex items-center">
+            <span
+                x-text="fileName"
+                class="mt-1 mr-2 ml-2 text-sm text-slate-200"
```

Comment:
```suggestion
                class="text-sm text-slate-200"
```

these should not be needed